### ISSUE TITLE
Handling inexisting users

### ISSUE THREAD

**Issue by Fraifrai** [jornane/mwSimpleSamlAuth#28]
Re,

Using $wgSamlCreateUser to true is OK but using false, unknown users get such an internal error :+1: 

```
MediaWiki internal error.

Original exception: [709656f6] /mediawiki/index.php/Accueil Exception from line 820 of /var/www/mediawiki/includes/skins/SkinTemplate.php: SkinTemplate::makeTalkUrlDetails given invalid pagename Utilisateur:
```

**Comment by jornane** [comment]
Strange, I need to test that…

**Comment by Fraifrai** [comment]
If the user do not exist, it tries ti render a standard page instead of providing an error message.

**Comment by Fraifrai** [comment]
I got it : the loadUser function returns a string if the user doesn't exist, but this string is not used in the calling code so that the extension tries to continue with a NULL id user.

**Comment by jornane** [comment]
Fixed in commit 4499700.
